senders/pushover/pushover.go

```diff
@@ -91,7 +95,17 @@ func (sender *Sender) buildMessage(events moira.NotificationEvents, throttled bo
 }
 
 func (sender *Sender) buildTitle(events moira.NotificationEvents, trigger moira.TriggerData) string {
-	return fmt.Sprintf("%s %s %s (%d)", events.GetSubjectState(), trigger.Name, trigger.GetTags(), len(events))
+	title := fmt.Sprintf("%s %s %s (%d)", events.GetSubjectState(), trigger.Name, trigger.GetTags(), len(events))
+	tags := 1
+	for len(title) > titleLimit-30 {
```

Comment:
We should use ```len([]rune(title))``` instead ```len(title)``` because ```len(title)``` returns size in bytes but we need to check symbols len

Reply:
Oh, you just should not use ```titleLimit-30```. Only ```titleLimit``` because in your case you build title in every cycle, but in other or sender's we reverse symbos for complete message.